frontend/src/features/admin-form/responses/IndividualResponsePage/mutations.ts

```diff
@@ -79,3 +82,39 @@ export const useMutateDownloadAttachments = () => {
 
   return { downloadAttachmentMutation, downloadAttachmentsAsZipMutation }
 }
+
+/**
+ * Converts address field info as Single Responses into single-line string for response page display
+ * String manipulation done on frontend in DecryptRow
+ * since backend requires inputs to be single responses for info to be separate csv columns
+ *
+ * responses format:
+ * ["blockNumber_161","streetName_BUKIT BATOK STREET 11","buildingName_","levelNumber_","unitNumber_","postalCode_650161"]
```

Comment:
We can use [`jsdocs`](https://jsdoc.app/tags-param) to annotate input format.

```suggestion
 * @param responses ["blockNumber_161","streetName_BUKIT BATOK STREET 11","buildingName_","levelNumber_","unitNumber_","postalCode_650161"]
```

Reply:
We can also annotate [`return`](https://jsdoc.app/tags-returns) values

Reply:
👍

Reply:
added this in the function which i moved to the shared util
```
 * @param responses ["161","BUKIT BATOK STREET 11","","01","02","650161"]
 * @return ["161 BUKIT BATOK STREET 11, #01-02, SINGAPORE 650161"]```